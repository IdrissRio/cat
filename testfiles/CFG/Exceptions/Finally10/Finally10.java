public class Finally10 {
  void m() {
    try {
      int a = 0;
      try {
        if (a > 0)
          return;
      } finally {
        int b;
      }
      int c;
    } finally {
      int d;
    }
    int e;
  }
}