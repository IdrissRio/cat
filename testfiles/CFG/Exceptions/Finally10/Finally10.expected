NodeName: Entry[15] Successors: [Exit[16]]
NodeName: Exit[16] Successors: []
NodeName: Entry[7] Successors: [Exit[8]]
NodeName: Exit[8] Successors: []
NodeName: Entry[29] Successors: [IntegerLiteral[49]]
NodeName: IntegerLiteral[49] Successors: [VariableDeclarator[46]]
NodeName: VariableDeclarator[46] Successors: [VarAccess[55]]
NodeName: VarAccess[55] Successors: [IntegerLiteral[56]]
NodeName: IntegerLiteral[56] Successors: [GTExpr[54]]
NodeName: GTExpr[54] Successors: [UncheckedExceptions[58], UncheckedExceptions[98]]
NodeName: UncheckedExceptions[58] Successors: [VariableDeclarator[69]]
NodeName: VariableDeclarator[69] Successors: [VariableDeclarator[81]]
NodeName: VariableDeclarator[81] Successors: [ReturnStmt[57], Exit[36]]
NodeName: ReturnStmt[57] Successors: [Exit[36]]
NodeName: Exit[36] Successors: []
NodeName: UncheckedExceptions[98] Successors: [VariableDeclarator[95]]
NodeName: VariableDeclarator[95] Successors: [VariableDeclarator[104], VariableDeclarator[116]]
NodeName: VariableDeclarator[104] Successors: [UncheckedExceptions[119]]
NodeName: UncheckedExceptions[119] Successors: [VariableDeclarator[116]]
NodeName: VariableDeclarator[116] Successors: [VariableDeclarator[125], Exit[36]]
NodeName: VariableDeclarator[125] Successors: [Exit[36]]

NodeName: Exit[16] Predecessor: [Entry[15]]
NodeName: Entry[15] Predecessor: []
NodeName: Exit[8] Predecessor: [Entry[7]]
NodeName: Entry[7] Predecessor: []
NodeName: Exit[36] Predecessor: [ReturnStmt[57], VariableDeclarator[81], VariableDeclarator[116], VariableDeclarator[125]]
NodeName: ReturnStmt[57] Predecessor: [VariableDeclarator[81]]
NodeName: VariableDeclarator[81] Predecessor: [VariableDeclarator[69]]
NodeName: VariableDeclarator[69] Predecessor: [UncheckedExceptions[58]]
NodeName: UncheckedExceptions[58] Predecessor: [GTExpr[54]]
NodeName: GTExpr[54] Predecessor: [IntegerLiteral[56]]
NodeName: IntegerLiteral[56] Predecessor: [VarAccess[55]]
NodeName: VarAccess[55] Predecessor: [VariableDeclarator[46]]
NodeName: VariableDeclarator[46] Predecessor: [IntegerLiteral[49]]
NodeName: IntegerLiteral[49] Predecessor: [Entry[29]]
NodeName: Entry[29] Predecessor: []
NodeName: VariableDeclarator[116] Predecessor: [UncheckedExceptions[119], VariableDeclarator[95]]
NodeName: UncheckedExceptions[119] Predecessor: [VariableDeclarator[104]]
NodeName: VariableDeclarator[104] Predecessor: [VariableDeclarator[95]]
NodeName: VariableDeclarator[95] Predecessor: [UncheckedExceptions[98]]
NodeName: UncheckedExceptions[98] Predecessor: [GTExpr[54]]
NodeName: VariableDeclarator[125] Predecessor: [VariableDeclarator[116]]
